tests/v2/validation/provisioning/k3s/README.md

```diff
@@ -348,6 +348,17 @@ These tests utilize Go build tags. Due to this, see the below examples on how to
 
 `gotestsum --format standard-verbose --packages=github.com/rancher/rancher/tests/v2/validation/provisioning/k3s --junitfile results.xml -- -timeout=60m -tags=validation -v -run "TestK3SPSACTTestSuite$"`
 
+### Hardened Custom Cluster
+This will provision a hardened custom cluster that runs across the following CIS scan profiles:
+1. `k3s-cis-1.8-profile-hardened`
+2. `k3s-cis-1.8-profile-permissive`
+
+You would use the same config that you setup for a custom cluster to run this test.
+
+These tests utilize Go build tags. Due to this, see the below examples on how to run the tests:
```

Comment:
I would link to the normal custom cluster section so that users know its required too. 

Also, mentioned in the other PR but can you list supported OS? I think its just ubuntu but we should document it.

Reply:
also do you think it would be good to link to the hardening guide, or does that link change too often?

Reply:
I wouldn't want to link the hardening guide personally. I mentioned in the shepherd PR, but I am not sure of the supported OS. I don't think that sets a good precedent, that implies that it's not OS-agnostic, but it is.

Reply:
Maybe a compromise here is I list that the AMI must have `sudo` capabilities? It will just be another requirement to bring, but we call it out. I can add that to the READMEs.

Reply:
Resolved